### DISCUSSION TITLE
Fedora libssl not found, openssl is installed

### DISCUSSION THREAD

**Discussion by NicoAICP** [unoplatform/uno#9349] [Q&A]
Hello there,

i have made an App with UNO, that already works on ubuntu without issues.
i wanted to test it in fedora, but when starting i get the issue that libssl is not found. 
Since i am having to use DNF instead of pacman or apt the package name is openssl. 
What should i do to get it to open? 
![image](https://user-images.githubusercontent.com/24915669/180668734-f28a2ad4-c295-4629-8c96-dbf31a162ef3.png)
Rough Translation of second window:
The package openssl is already installed 
No package found called libssl

**Comment by jeromelaban** [comment]
This looks like https://github.com/dotnet/runtime/issues/27113#issuecomment-431699520 is a workaround you can try.
